test/parallel/test-buffer.js

```diff
@@ -1179,3 +1179,7 @@ Buffer.poolSize = ps;
 assert.throws(function() {
   Buffer(10).copy();
 });
+
+assert.throws(function() {
+  new Buffer();
+}, 'Unable to get property');
```

Comment:
Why not use a regex here (i.e. `/must start with number, buffer, array or string/`) rather than what you already have? AFAICT it doesn't seem to actually test the error message.